.gitmodules

```diff
@@ -10,7 +10,7 @@
 [submodule "lib/se050"]
 	path = lib/se050
 	url = https://github.com/rddl-network/seal-edge-apdu.git
-	branch = main
+	branch = fasagidag/adding_secp
```

Comment:
I guess this branch should be merged to main ant some point?

Reply:
I am aware of this, while I was working on this branch, I did not merge it into main in case you would compile se050. Should I merge the seal-edge-apdu fasagidag/adding_secp branch to main now?  Or after this PR closes?

Reply:
I would merge it to main now so this repository also relies on the main branch of the submodule.